Design.md

```diff
@@ -136,46 +121,88 @@ This API will update the existing storage tenant with the provided `cd_tenant_id
 
 1. Vault `update-account` api will be called using vaultclient with the provided `tenantID` as `AccountID` and `tenant` properties
 
+### `SetupAssumeRole` subroutine
+1. Use vaultclient to call `generate-account-access-key` with `expirationTime` for the account.
+1. Create a role using the generated access key with:
+    1. Role name: `osis` (So role-arn can be generated in the format `arn:aws:iam::[account-id]:role/osis`)
+    1. Trust policy:
+   ```json
+    {
+        "Version": "2012-10-17",
+        "Statement": [{
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "service-name"
+            },
+            "Action": "sts:AssumeRole"
+            }]
+    }
+   ```
+    * **Note:** 
+        * _This role will be visible and can be edited by the root account user (or) the IAM user with the right permissions (iam:list-roles, iam:deleteRoles...). **Editing this role is strictly not suggested. If edited, this role must be manually deleted**_
```

Comment:
```suggestion
        * _This role will be visible to and modifiable by the root account user or by an IAM user with correct permissions (iam:list-roles, iam:deleteRoles...). **Do not edit this role. If this role is edited, it must be deleted**_
```

This language simplifies the instruction, but I don't see the point of juxtaposing how to edit with a warning not to edit. Is there more to it than this?